extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/app.js

```diff
@@ -105,11 +105,15 @@ angular.module('PaperUI', [ 'PaperUI.controllers', 'PaperUI.controllers.control'
             redirectTo : '/control'
         });
     }
+    $mdDateLocaleProvider.shortMonths = [ 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec' ];
+    if (window.localStorage.getItem('paperui.language') == 'de') {
+        $mdDateLocaleProvider.shortMonths = [ 'Jan', 'Feb', 'Mär', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Dez' ];
+    }
     $mdDateLocaleProvider.formatDate = function(date) {
         if (!date) {
             return null;
         }
-        return (date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate());
+        return (date.getDate() + '.' + (date.getMonth() + 1) + '.' + date.getFullYear());
```

Comment:
Shouldn't we use `window.localStorage.getItem('paperui.language')` here, too?
e.g. to differ between 31.12.2015 for german and 12/31/2015 for english and a common fallback for others (but ATM we support only german and english - correct?